app/views/layouts/_header.html.erb

```diff
@@ -1,3 +1,5 @@
+<link rel="stylesheet" href="font-awesome-animation.min.css">
```

Comment:
I don't think this is needed?

Reply:
Oops my bad ! I initially integrated it with the fontawesome CSS but this seemed to work better. Will remove that line right away